manta-pay/src/simulation/ledger/mod.rs

```diff
@@ -452,9 +484,18 @@ impl ledger::Write<Vec<TransferPost>> for LedgerConnection {
     }
 }
 
-impl PublicBalanceOracle for LedgerConnection {
+impl PublicBalanceOracle<Config> for LedgerConnection {
     #[inline]
-    fn public_balances(&self) -> LocalBoxFuture<Option<AssetList>> {
+    fn public_balances(
+        &self,
+    ) -> LocalBoxFuture<
+        Option<
+            AssetList<
+                <Config as transfer::Configuration>::AssetId,
+                <Config as transfer::Configuration>::AssetValue,
```

Comment:
For all of these kinds of things just make an alias in the `crate::config` file for `AssetId` and `AssetValue` and call them instead.